{{cookiecutter.project_name}}/Makefile

```diff
@@ -183,6 +183,8 @@ fix: depends-dev
 	$(PEP8RADIUS) --docformatter --in-place
 
 # Testing ######################################################################
+
+TIMESTAMP := $(shell date +%s)
```

Comment:
Because the `TIMESTAMP` is a seed, I'd like it to be a bit more random. I'm going to keep this for now and look at getting builds running for Windows: https://github.com/jacebrowning/template-python/issues/92